### DISCUSSION TITLE
好きな国を教えて下さい! Tell me your favorite country!助けて！！！

### DISCUSSION THREAD

**Discussion by 552bdbce** [NAIST-SE/AttractingContributors#147] [General]
https://github.com/hiroki-ishimaru/AttractingContributors

**Comment by MatsumuraTaiki** [comment]
Added, please help me too
https://github.com/MatsumuraTaiki/AttractingContributors-Actors/edit/main/README.md

**Reply by nagata-atsuki** [reply]
added!
Please help me
https://github.com/nagata-atsuki/AttractingContributors-Athlete
